### ISSUE TITLE
[BUG]: Win 11 "takeown /d y" parameter is language dependent

### ISSUE THREAD

**Issue by arman-ku** [undergroundwires/privacy.sexy#149]
win 11 10.0.22000.778

some options use `takeown` with `/d y` parameters. The y is language dependent however.
For example, i installed a german build, **so the "y" would be a "j" for "ja".**

Code example:

> :: ----------------------------------------------------------
> :: ----------------Clear main telemetry file-----------------
> :: ----------------------------------------------------------
> echo --- Clear main telemetry file
> if exist "%ProgramData%\Microsoft\Diagnosis\ETLLogs\AutoLogger\AutoLogger-Diagtrack-Listener.etl" (
>     takeown /f "%ProgramData%\Microsoft\Diagnosis\ETLLogs\AutoLogger\AutoLogger-Diagtrack-Listener.etl" /r /d y
>     icacls "%ProgramData%\Microsoft\Diagnosis\ETLLogs\AutoLogger\AutoLogger-Diagtrack-Listener.etl" /grant administrators:F /t
>     echo "" > "%ProgramData%\Microsoft\Diagnosis\ETLLogs\AutoLogger\AutoLogger-Diagtrack-Listener.etl"
>     echo Clear successful: "%ProgramData%\Microsoft\Diagnosis\ETLLogs\AutoLogger\AutoLogger-Diagtrack-Listener.etl"
> ) else (
>     echo "Main telemetry file does not exist. Good!"
> )

will throw:

`Error: Invalid syntax. The 'y' value cannot be used for the '/d' option.`

**Comment by undergroundwires** [comment]
Thank you @arman-ku. I can reproduce this on german versions of Windows.

Since this issue was raised (around the 0.12.X releases), I've already removed most instances of `takeown` commands, including the script you shared.However, there's still [one remaining usage](https://github.com/undergroundwires/privacy.sexy/blob/e95b2ba2179e40c0033a51b0087871dbfdc32d78/src/application/collections/windows.yaml#L14379).

Here's a summary of the options considered:

- ❌ Enumerating all languages for their "yes" equivalents. [Yes and No in many languages list](https://www.omniglot.com/language/phrases/yesnomaybe.htm) list is too extensive and impractical 🙂
- ❌ Setting the current culture before executing (`[Threading.Thread]::CurrentThread.CurrentCulture`). This doesn't work.
- ❌ Leveraging the side-effect of the `copy` command output to determine the "yes" equivalent, as suggested in [this ServerFault post](https://serverfault.com/a/1004687). While reliable, it's overly complex.
- ❌ Parsing the `takeown /?` help documentation. This approach proved unreliable as the documentation doesn't always accurately reflect command variations due to incomplete i18n translations.
- ✅ Use [choice](https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/choice) Windows utility to get available choices and return the first item. Tested on English: (`Y`) and German (`J`).

The most effective solution I found is to use the `choice` Windows command, which reliably determines the current language's equivalent for "yes". This approach has been tested successfully in both English (`Y`) and German (`J`).

I'm planning to implement this solution using the `choice` command in the upcoming patches.
